vendor/assets/javascripts/components/accordion.js

```diff
@@ -1,14 +1,14 @@
-/*! UIkit 2.26.4 | http://www.getuikit.com | (c) 2014 YOOtheme | MIT License */
+/*! UIkit 2.27.2 | http://www.getuikit.com | (c) 2014 YOOtheme | MIT License */
 (function(addon) {
     var component;
 
-    if (window.UIkit) {
-        component = addon(UIkit);
+    if (window.UIkit2) {
```

Comment:
see my note on the PR itself, this would introduce a breaking change and is inconsistent with the YooTheme source